synapse/config/workers.py

```diff
@@ -425,6 +368,50 @@ def _should_this_worker_perform_duty(
         # (By this point, these are either the same value or only one is not None.)
         return bool(new_option_should_run_here or legacy_option_should_run_here)
 
+    def _worker_names_performing_this_duty(
+        self,
+        config: Dict[str, Any],
+        legacy_option_name: str,
+        legacy_app_name: str,
+        modern_instance_map_name: str,
+    ) -> List[str]:
+        """
+        retrieves the names of the workers handling a given duty, by either legacy option or instance_map
+        config: settings read from yaml.
+        legacy_option_name: the old way of enabling options. e.g. 'start_pushers'
+        legacy_app_name: The historical app name. e.g. 'synapse.app.pusher'
+        modern_instance_map_name: the string name of the new instance_map. e.g. 'pusher_instances'
+        """
+
+        legacy_option = config.get(legacy_option_name, True)
+
+        worker_instance_map = config.get(modern_instance_map_name)
```

Comment:
Calling this `worker_instances` I think is clearer since it is a list, not a map.

```suggestion
        worker_instances = config.get(modern_instance_map_name)
```

Reply:
I just went ahead and changed all references to "map" and similar. It is a list, so it should reflect that. Fixed in 47ceef6